### DISCUSSION TITLE
How to check if crashlog was submitted to EarTrumpet team? I had a 2nd app crash within 4 weeks just now...

### DISCUSSION THREAD

**Discussion by D4n2021** [File-New-Project/EarTrumpet#1390] [General]
Where is the crash data being sent to, where can I see the log to check if it really was sent? 

![grafik](https://github.com/File-New-Project/EarTrumpet/assets/20766261/25dd279c-81dd-4975-a2d9-6ba5c3791822)

**Comment by riverar** [comment]
@D4n2021 Nothing you can really check on your end. We use Bugsnag; if you paste your .NET exception from Event Viewer (or if you feel comfortable sharing your PC name) I might be able to find it.
